src/oc_erchef/apps/data_collector/src/data_collector.erl

```diff
@@ -24,8 +24,9 @@
 -module(data_collector).
```

Comment:
Given that a lot of the meat of this ended up in the oc_chef_wm app, I wonder if this really needs to be its own app.  However, that feels like a refactor that can easily be made down the road if we want so this layout is fine by me for now.